### PR TITLE
Model depth for Maxwell 2D transient analysis is now a property with getter/setter.

### LINKED ISSUE #753: Method missing: set model depth in Maxwell 2D
Dear All,

I could find the method: Maxwell2d.get_model_depth()
 but not the way to set the model depth for 2D transient analysis.
In the GUI it is under:

Maxwell2D-> Design Settings->Model Settings
Thank you very much,

Tiziana

### PR DESCRIPTION
**Pull request by MaxJPRey**
Model depth for Maxwell 2D transient analysis is now a property with getter/setter.

Fix #753 .

### CODE REVIEW

File: pyaedt/maxwell.py [hunk 1]
```diff
@@ -1159,18 +1159,56 @@ def xy_plane(self):
...
+        """Get model depth.
+
+        Example
+        -------
+
+        Set and get the model depth for a 2D Maxwell analysis.
+
+        >>> from pyaedt import Maxwell2d
+        >>> maxwell_2d = Maxwell2d()
+        >>> maxwell_2d.model_depth="90 mm"
```
Comment (akaszynski) [inline pyaedt/maxwell.py:1173]:
```suggestion
        >>> maxwell_2d.model_depth = "90 mm"
```

File: _unittest/test_27_Maxwell2D.py [hunk 2]
```diff
@@ -108,3 +108,13 @@ def test_16_enable_inductance_computation(self):
...
+
+    # @pyaedt_unittest_check_desktop_error
+    # def test_17_enable_inductance_computation(self):
+    #     # Set a new value for model depth
+    #     self.aedtapp.model_depth = "90 mm"
+    #     assert self.aedtapp.model_depth == 0.09
+    #     self.aedtapp.model_depth = "1.3"
+    #     assert self.aedtapp.model_depth == 1.3
+    #     # Generate design data
+    #     assert self.aedtapp.generate_design_data()
```
Comment (maxcapodi78) [inline _unittest/test_27_Maxwell2D.py:120]: If not needed we can remove this comment

**Review by maxcapodi78 — APPROVED** [review]
LGTM

File: _unittest/test_27_Maxwell2D.py (hunk 2 above)
Comment (MaxJPRey) [reply _unittest/test_27_Maxwell2D.py]: Done. I will reinstate the test when the project properties issue is fixed.

File: pyaedt/maxwell.py [hunk 3]
```diff
@@ -1161,46 +1161,22 @@ def xy_plane(self, value=True):
...
+        if "ModelDepth" in self.design_properties:
+            value_str = self.design_properties["ModelDepth"]
+            try:
+                a = float_units(value_str)
+            except:
+                a = self.variable_manager[value_str].value
+            finally:
+                return a
+        else:
+            return None
```
Comment (akaszynski) [inline pyaedt/maxwell.py:1175]: This seems redundant here.

**Pull request merged**